binchihua/src/orch/orch.c

```diff
@@ -1,16 +1,20 @@
+#include <stdio.h>
```

Comment:
You re-order the includes to have first the external libraries and second the local ones (separating by an empty line). Is this a best practice? If so, we should do that as well for all the other files. I did the same with the opposite order.

Reply:
systemd puts the system headers first, and the local headers after that.  For example, in systemd/src/basic/conf-files.c:

```
#include <errno.h>
#include <stdarg.h>
#include <stdio.h>
#include <stdlib.h>

#include "chase-symlinks.h"
#include "conf-files.h"
#include "constants.h"
#include "dirent-util.h"
#include "fd-util.h"
#include "hashmap.h"
...
```

Yea, we might want to do this for better readability.

Reply:
Sounds good. I'll add applying this to the other source files to the issue that I'll create for https://github.com/engelmi/orch/pull/55#discussion_r1065878054